src/TensorValues/Operations.jl

```diff
@@ -220,8 +246,40 @@ function inner(a::SymFourthOrderTensorValue{D},b::MultiValue{Tuple{D,D}}) where
   inner(a,symmetric_part(b))
 end
 
+# a_i = b_ijk*c_jk
+@generated function inner(a::A, b::B) where {A<:MultiValue{Tuple{D1,D2,D3}},B<:MultiValue{Tuple{D2,D3}}} where {D1,D2,D3}
```

Comment:
This is not an inner product. This is a double contraction, isn't it?

Reply:
Yes. I was trying to follow the convention that has been used in the rest of the file. I'll update these with the above notation